.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,15 @@ GitHub issue resolved: # <!-- add a Github issue number here, e.g #123. This lin
 
 Briefly describe the changes proposed in this PR:
 
-* 
-* 
-* 
+<!-- short description of your change goes here -->
+
+---- 
+PR Author Checklist: 
+
+ - [ ] my pull request is [self-contained](https://rdf4j.org/documentation/developer/merge-strategy/#self-contained-changes-pull-requests-and-commits)
+ - [ ] I've added tests for the changes I made
+ - [ ] every commit message starts with the issue number (GH-xxxx) followed by a meaningful description of the change
+ - [ ] every commit has been [signed off](https://stackoverflow.com/questions/1962094/what-is-the-sign-off-feature-in-git-for)
```

Comment:
From having added a checklist like this in one of my previous projects I don't belive that it actually works as a checklist. We can always try, but I think it would work just as well as a regular list.

Reply:
Yeah, agreed, I've used it before as well and it never really works as a proper list to check things off. My intent here is not so much to force people to actually tick the boxes, more as a list of reminders - using checkboxes to render to make it (hopefully) a little more obvious that these are actionable items.

If people tick them, great. If not, no harm done either.